pkg/registry/deployment.go

```diff
@@ -181,11 +183,22 @@ func GenerateDeployment(cr *registryv1alpha1.DevfileRegistry, scheme *runtime.Sc
 					},
 				},
 			},
+			Env: []corev1.EnvVar{
+				{
+					Name:  "ANALYTICS_WRITE_KEY",
+					Value: cr.Spec.Telemetry.RegistryViewerWriteKey,
+				},
+				{
+					Name: "DEVFILE_REGISTRIES",
+					Value: fmt.Sprintf(`[{\"name\":\"Community\",\"url\":\"http://localhost:8080\",\"fqdn\":\"http://%s.%s\"}]`,
+						IngressName(cr.Name), cr.Spec.K8s.IngressDomain),
```

Comment:
Are you planning to update the fqdn to `cr.Status.URL` in a different PR?

Reply:
@kim-tsao Going to rebase https://github.com/devfile/registry-operator/pull/29 on these changes and change it there.